src/components/atoms/CloseButton/CloseButton.tsx

```diff
@@ -3,12 +3,12 @@ import { XMarkIcon } from '@heroicons/react/24/outline';
 export const CloseButton = ({ onClick }: { onClick: () => void }) => {
     return (
         <button
-            className='flex h-8 w-8 items-center justify-center rounded-full border-2 border-neutral-3 hover:border-white-30'
+            className='flex h-5 w-5 items-center justify-center rounded-full border-[1.25px] border-neutral-3 hover:border-white-30 tablet:h-7 tablet:w-7 tablet:border-[1.75px]'
```

Comment:
Style for close button on tablet is here: https://www.figma.com/design/YIBsk1ihFbPlDb8XldGj1T/SF-Design-System?node-id=691-11968&m=dev